html/janus.js

```diff
@@ -1941,8 +1941,9 @@ function Janus(gatewayCallbacks) {
 					trackMutedTimeoutId = setTimeout(function() {
 						Janus.log('Removing remote track');
 						// Notify the application the track is gone
-						let transceiver = config.pc.getTransceivers().find(
-							t => t.receiver.track === ev.target);
+						let transceivers = config.pc ? config.pc.getTransceivers() : null;
+						let transceiver = transceivers ? transceivers.find(
+							t => t.receiver.track === ev.target) : null;
```

Comment:
since `pc.getTransceivers()` will always return an array, everything could be simplified in a form more similar to the original code

```js
let transceiver = config.pc ? config.pc.getTransceivers().find(t => t.receiver.track === ev.target) : null;
```